table/src/main/java/com/bazaarvoice/emodb/table/db/astyanax/AstyanaxTableDAO.java

```diff
@@ -1386,4 +1393,16 @@ public void clearStashTokenRangeSnapshot(String stashId) {
         checkState(_stashTableDao != null, "Call only valid in Stash mode");
         _stashTableDao.clearTokenRanges(stashId);
     }
+
+    @VisibleForTesting
+    static boolean isTableBlacklisted(String table, String placement, Condition blackListTableCondition) {
+        Condition thisTableCondition = Conditions.and(
+                Conditions.intrinsic(Intrinsic.TABLE, Conditions.equal(table)),
+                Conditions.intrinsic(Intrinsic.PLACEMENT, Conditions.equal(placement)));
+        if (SubsetEvaluator.isSubset(thisTableCondition, blackListTableCondition)) {
+            return true;
+        }
+        return false;
```

Comment:
The above implementation isn't doing exactly what we want.  We're trying to determine if the table matches the condition in `blacklistTableCondition`.  The correct implementation is similar to https://github.com/bazaarvoice/emodb/blob/master/databus/src/main/java/com/bazaarvoice/emodb/databus/core/SubscriptionEvaluator.java#L57 .  The sequence would be something like:

1. Get the `Table` named "table".  This was already instantiated on line 1366, so you could just pass it to this function.
2. Get the table's attributes and create a `TableFilterIntrinsics`
3. Use `ConditionEvaluator.eval()` with the `blackListTableCondition` and the two above objects, and if it returns true then the table is blacklisted.